From 46c022ed818ef55a04a0bed68c2c4b0153ce43eb Mon Sep 17 00:00:00 2001
From: Jonghwa Lee <jonghwa3.lee@samsung.com>
Date: Thu, 31 Oct 2013 11:20:56 +0900
Subject: [PATCH 0899/1302] charger-manager: Fix unit of temperature to deci
 centigrade.

Fix temperature unit to follow the standard as document describes.

Signed-off-by: Jonghwa Lee <jonghwa3.lee@samsung.com>
Signed-off-by: MyungJoo Ham <myungjoo.ham@samsung.com>
---
 arch/arm/boot/dts/exynos4412-slp_pq.dts | 6 +++---
 drivers/power/charger-manager.c         | 8 ++++----
 2 files changed, 7 insertions(+), 7 deletions(-)

diff --git a/arch/arm/boot/dts/exynos4412-slp_pq.dts b/arch/arm/boot/dts/exynos4412-slp_pq.dts
index 9f98e32..86c6e5f 100644
--- a/arch/arm/boot/dts/exynos4412-slp_pq.dts
+++ b/arch/arm/boot/dts/exynos4412-slp_pq.dts
@@ -1290,10 +1290,10 @@
 
 		cm-battery-stat = <3>;
 
-		cm-battery-cold = <5000>;
+		cm-battery-cold = <50>;
 		cm-battery-cold-in-minus;
-		cm-battery-hot = <65000>;
-		cm-battery-temp-diff = <15000>;
+		cm-battery-hot = <650>;
+		cm-battery-temp-diff = <150>;
 
 		cm-num-chargers = <1>;
 		cm-chargers = "max77693-charger";
diff --git a/drivers/power/charger-manager.c b/drivers/power/charger-manager.c
index 6974691..373281f 100644
--- a/drivers/power/charger-manager.c
+++ b/drivers/power/charger-manager.c
@@ -1462,10 +1462,10 @@ err:
 	return ret;
 }
 
-/* Every temperature units are in milli centigrade */
-#define CM_DEFAULT_TEMP_ALERT_DIFF	10000
-#define CM_DEFAULT_TEMP_ALERT_MAX	127000
-#define CM_DEFAULT_TEMP_ALERT_MIN	(-127000)
+/* Every temperature units are in decii centigrade */
+#define CM_DEFAULT_TEMP_ALERT_DIFF	100
+#define CM_DEFAULT_TEMP_ALERT_MAX	1270
+#define CM_DEFAULT_TEMP_ALERT_MIN	(-1270)
 
 static int cm_default_get_temp(struct charger_manager *cm)
 {
-- 
1.8.3.2

